### DISCUSSION TITLE
Time zone does not work under Mongo.

### DISCUSSION THREAD

**Discussion by Lydanne** [mikro-orm/mikro-orm#2345] [Q&A] [answered]
**Describe the bug**
Time zone does not work under Mongo.

![image](https://user-images.githubusercontent.com/39021696/140271651-a64ed388-06ab-437a-bb60-4d8a544f426a.png)

![image](https://user-images.githubusercontent.com/39021696/140271622-f73eed92-75ef-4d26-9539-3174174e2160.png)

**Expected behavior**
I want the time to be -8:00 on writes and +8:00 on reads

**Versions**

| Dependency | Version |
| - | - |
| node | 14 |
| typescript | ? |
| mikro-orm | 4.5.9 |
| your-driver | mongo |

**Comment by Lydanne** [comment]
My expected createdAt: "2021-11-04T015:06:08.368Z"

**Comment by B4nan** [comment, marked as answer]
That configuration is for SQL drivers, and it is there to allow setting timezone on dates _without_ timezone columns (e.g. datetimes in mysql). In mongo the date always has the TZ information, you don't really need it. Idk if there is something similar in mongo driver, haven't found anything right now - if there is some support in the connector, you can use `driverOptions` to pass it there. 

What you are asking for does not make much sense to me, that is not how the flag works in SQL drivers. Also note that in general using TZ offset is wrong too, as it does not handle DST.
